src/lib/utils/invite.ts

```diff
@@ -2,7 +2,19 @@ import * as ob from 'urbit-ob';
 
 export const generateUrl = (ticket: string, planet: number) => {
   if (ticket && planet) {
-    return `urb.it/#${ticket.slice(1)}-${ob.patp(planet).slice(1)}`;
+    return `bridge.urbit.org/#${ticket.slice(1)}-${ob.patp(planet).slice(1)}`;
+  } else if (planet) {
+    return ob.patp(planet);
+  }
+
+  return 'No Ticket or Ship';
+};
+
+export const generateUrlAbbreviation = (ticket: string, planet: number) => {
+  if (ticket && planet) {
+    return `bridge.urbit.org/#${ticket.slice(1, 7)}...${ob
```

Comment:
Optionally, would be nice to use the current host in the URL, especially if we also plan to deploy an instance to a mainnet staging environment.

```suggestion
    return `${location.host}/#${ticket.slice(1, 7)}...${ob
```

Reply:
Good call, will put this in